opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/validation/Xs2aValidator.java

```diff
@@ -30,11 +33,16 @@
 
     private final Validator validator;
 
-    public void validate(DelegateExecution exec, Object... dtosToValidate) {
+    public void validate(DelegateExecution exec, Xs2aContext context, Object... dtosToValidate) {
         Set<ConstraintViolation<Object>> allErrors = new HashSet<>();
 
+        Set<String> fieldsToIgnore = getFieldsToIgnoreValidate(context);
+
         for (Object value : dtosToValidate) {
-            Set<ConstraintViolation<Object>> errors = validator.validate(value);
+            Set<ConstraintViolation<Object>> errors = validator.validate(value)
+                    .stream()
+                    .filter(f -> !fieldsToIgnore.contains(findInfoOnViolation(f).ctx().value().name()))
```

Comment:
Better to use enums for field codes - ctx().value() is enum - FieldCode

Reply:
To have access to FieldCode enum from facade I have to move it to shared or create new FieldCodeDto enum in shared module

Reply:
So it must be shared

Reply:
ok